.github/workflows/tests.yml

```diff
@@ -26,6 +26,6 @@ jobs:
         run: |
           source $CONDA/etc/profile.d/conda.sh
           conda activate taranis_env
-          poetry install
+          python -m pip install .
           taranis analyze-schema -i  test/MLST_listeria -o analyze_schema_test  --cpus 1 --output-allele-annot --remove-no-cds --remove-duplicated --remove-subset
```

Comment:
please add the test for the reference_alleles functionality

Reply:
I want  to finish coding first, as I  would like to start using pytest for testing